examples/stats/plot_encoding_model.py

```diff
@@ -0,0 +1,105 @@
+"""
+========================================
+Regression on continuous and event-based data)
+========================================
+
+Neat description. Oooo so cool.
+
+rERPs are described in:
+Smith, N. J., & Kutas, M. (2015). Regression-based estimation of ERP
+waveforms: II. Non-linear effects, overlap correction, and practical
+considerations. Psychophysiology, 52(2), 169-189.
+"""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#          Jona Sassenhagen <jona.sassenhagen@gmail.de>
```

Comment:
Thanks, but this is probably massively overstating my contributions :)

Example seems cool (haven't looked at it all)